src/settings-ui/Settings.UI/Controls/ShortcutControl/ShortcutControl.xaml.cs

```diff
@@ -394,7 +406,12 @@ private void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                    hook.Dispose();
+                    if (hook != null)
+                    {
+                        hook.Dispose();
+                    }
+
+                    hook = null;
```

Comment:
Doesn't this makes more sense inside the if? Why should we set it if it's already `null`.